azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/AppServiceUtils.java

```diff
@@ -81,4 +86,27 @@ public static String convertPricingTierToString(final PricingTier pricingTier) {
         }
         return result;
     }
+
+    public static AppServicePlan getAppServicePlanByAppService(final WebAppBase webApp) {
+        return webApp.manager().appServicePlans().getById(webApp.appServicePlanId());
+    }
+
+    public static AppServicePlan updateAppServicePlan(final AppServicePlan appServicePlan,
+                                                      final PricingTier pricingTier,
+                                                      final Log log) throws MojoExecutionException {
+        if (appServicePlan == null) {
+            throw new MojoExecutionException(SERVICE_PLAN_NOT_FOUND);
+        }
+        log.info(String.format(UPDATE_APP_SERVICE_PLAN));
```

Comment:
"Updating app service plan" is also proper for functions?

Reply:
Function is also a kind of app service deployed in app service plan, so i think it could be used for function too.

Reply:
use 'Updating service plan' instead?

Reply:
Portal also uses `App Service plan`, so it should be fine.
![image](https://user-images.githubusercontent.com/12445236/70013102-7a17db80-15b1-11ea-9a3a-e88a33459725.png)